source/documentation/deploying_services/redis.md

```diff
@@ -8,32 +8,34 @@ Redis is an open source in-memory data store that can be used as a database cach
 
 To set up a Redis service:
```

Comment:
Possibly beyond the scope of this PR: There is a lot of repetition in rows 5-9, plus no text between the H2 and H3.

Reply:
Driven by accessibility header work. I needed to categorise the H3s.